components/builder-api/src/depot/server.rs

```diff
@@ -945,12 +954,11 @@ fn upload_package(req: &mut Request) -> IronResult<Response> {
             builder_flag,
         ) {
             Ok(_) => {
-                let mut response = Response::with((
-                    status::Created,
-                    format!("/pkgs/{}/download", package.get_ident()),
-                ));
+                let ident = package.get_ident();
```

Comment:
Would be nice not to change any of this code - just use package.get_ident() .. it will make life easier in rebase